### DISCUSSION TITLE
Joining across database files

### DISCUSSION THREAD

**Discussion by markpayneatwork** [duckdb/duckdb#4534] [General]
Hi,

I have two duckdb databases (as files on disk) containing tables that I would like to link together via a left_join i.e. joining table1 from db1 to table2 from db2. How can I best do this?

My current understanding is that ATTACH is not supported (yet?) as of 0.4.0. Are there plans to move in this direction? Or is there some smarter solution that I am not aware of coming up?

Are there any good workarounds that can be used in the meantime?

Best wishes,

Mark

**Comment by hannes** [comment]
You could write to parquet in the meantime, this is not yet supported, but its on our roadmap

**Reply by markpayneatwork** [reply]
Thanks for the reply. Is there documentation that you could point to regarding interactions with Parquet? I have seen it mentioned many times here, but can't seem to figure out how it would work....

**Reply by Alex-Monahan** [reply]
Check out the copy and export clauses!
https://duckdb.org/docs/data/parquet
